.changeset/new-eggs-admire.md

```diff
@@ -0,0 +1,23 @@
+---
+'sku': minor
+---
+
+Update `start` and `start-ssr` scripts to reuse an existing browser tab if available. 
```

Comment:
Technically this feature already exists, but we're just extending it to support more browsers.